main/src/ca/uwaterloo/flix/language/errors/WeederError.scala

```diff
@@ -17,6 +17,7 @@
 package ca.uwaterloo.flix.language.errors
 
 import ca.uwaterloo.flix.language.{CompilationMessage, CompilationMessageKind}
+import ca.uwaterloo.flix.language.ast.Name.QName
```

Comment:
For reasons lost to history, we always write Name.QName-- so QName should not be directly imported.